stacks/go/OWNERS

```diff
@@ -1,4 +1,11 @@
 # See the OWNERS docs: https://go.k8s.io/owners
 
 reviewers:
-- schultzp2020
+- elsony
+- johnmcollier
```

Comment:
You don't have to mention people that are in top-level `OWNERS` file again. 

The `OWNERS` file implements inheritance of sorts. If you have someone in top-level `OWNERS` file, they are automatically a reviewer/approver in everything.

This can make maintenance of `OWNERS` files easier.

Reply:
I didn't realize there were 3 levels of OWNERs.  It's the repo level [OWNER](https://github.com/devfile/registry/blob/main/OWNERS) that is responsible for approving/reviewing the integration tests.  I think we can keep that devfile specific @elsony .  At the overall stack level, there is a different [OWNERS](https://github.com/devfile/registry/blob/main/stacks/OWNERS) file.  For that one, we can add the representatives from the tools teams as reviewers in addition to the devfile team as @kadel  suggested.  I can then delete the OWNERs files from devfile owned stacks.